src/NumericalAlgorithms/DiscontinuousGalerkin/FluxCommunicationTypes.hpp

```diff
@@ -51,9 +51,9 @@ struct FluxCommunicationTypes {
 
```

Comment:
You can just delete this file, I forgot to do that when I finished refactoring this code. I can also put in a quick PR tomorrow